test/controllers/helper_methods_test.rb

```diff
@@ -0,0 +1,21 @@
+require 'test_helper'
+
+class ApiController < ActionController::Metal
+  include Devise::Controllers::Helpers
+end
+
+class HelperMethodsTest < ActionController::TestCase
+  tests ApiController
+
+  test 'includes Devise::Controllers::Helpers' do
+    assert @controller.class.ancestors.include?(Devise::Controllers::Helpers)
+  end
+
+  test 'does not respond_to helper_method' do
+    refute @controller.respond_to?(:helper_method)
+  end
+
+  test 'defines methods like current_user' do
+    assert @controller.respond_to?(:current_user)
+  end
```

Comment:
Please use the builtin methods: `assert_includes`, `assert_respond_to` and `refute_respond_to`. They provide nicer error messages in case of test failures.